src/github.com/getlantern/geolookup/geolookup.go

```diff
@@ -96,21 +98,16 @@ type Country struct {
 	} `maxminddb:"traits"`
 }
 
-// SetHTTPClient defines the client to use for executing HTTP requests.
-func SetHTTPClient(client *http.Client) {
-	httpClient = client
+// LookupCity looks up the given IP using a geolocation service and returns a
+// City struct. If an httpClient was provided, it uses that, otherwise it uses
+// a default http.Client.
+func LookupCity(ipAddr string, httpClient *http.Client) (*City, error) {
```

Comment:
Instead of maintaing a package-wide client, we just allow the caller to pass it in.